dump/parser.go

```diff
@@ -85,6 +88,25 @@ func Parse(r io.Reader, h ParseHandler, parseBinlogPos bool) error {
 			}
 		}
 
+		if !inTableParsing {
+			if m := tableStartExp.FindAllStringSubmatch(line, -1); len(m) == 1 {
+				inTableParsing = true
+			}
+		}
+
+		if inTableParsing {
+			if i := strings.IndexByte(line, byte(';')); i > 0 {
```

Comment:
maybe you want `strings.Contains`

And what's the purpose of this logic? what if the column comments contain `;`?

Reply:
resolved in [e91fc1f](https://github.com/go-mysql-org/go-mysql/pull/767/commits/e91fc1f23e4f15d5bbe30c3be88bf402beb78621)

Reply:
IndexByte may be faster than strings.Contains